src/TransitionGroup.js

```diff
@@ -49,7 +49,37 @@ const defaultProps = {
  * state machine for managing the mounting and unmounting of components over
  * time.
  *
- * Consider the example below using the `Fade` CSS transition from before.
+ * `<TransitionGroup>` clones its children with injected callbacks for the
+ * `<Transition>` component, so you should make sure to pass extra props through
+ * if your children are components that render a `<Transition>`.  For example:
+ *
+ * const duration = 300;
```

Comment:
you need to surround the code blocks in backtick blocks, like on github. in fact it's github style markdown here.